pkg/webservice/handlers.go

```diff
@@ -58,6 +58,11 @@ const GroupNameMissing = "Group name is missing"
 const ApplicationDoesNotExists = "Application not found"
 const NodeDoesNotExists = "Node not found"
 
+type outputConfig struct {
+	*configs.SchedulerConfig
+	Extra map[string]string `yaml:",omitempty" json:",omitempty"`
+}
```

Comment:
Can we move this struct to `webservice/dao/config_info`? so that https://github.com/apache/yunikorn-core/pull/664 can also use this. Also lets keep it as just `config`.